### ISSUE TITLE
some issues about R-Net and O-Net landmark faces training

### ISSUE THREAD

**Issue by OceanWong1991** [AITTSMD/MTCNN-Tensorflow#113]
Hi AITTSMD:
     Firstly, Thanks for your amazing project! when I read your code I find some different from the original paper:
    
- R-Net: We use the first stage of our framework to detect faces from WIDER FACE [24] to collect positives, negatives and part face **while landmark faces are detected from** CelebA [23].

- O-Net: Similar to R-Net to collect data but we use the first two stages of our framework to detect faces and collect data.

in my conception, the paper use landmark faces are detected from dataset, which different from your project, did you notice that?
@AITTSMD

**Comment by AITTSMD** [comment]
@OceanWong1991 Hi，yes! The original author uses Celeba dataset to train landmark part. However, when I used Celeba，I found some labels in Celeba were wrong. So I change the dataset.But the result is not that good. You can try to use Celeba dataset to train landmark part. The procedure is the same.

**Comment by OceanWong1991** [comment]
@AITTSMD Thaks for your replay! I use the Celeba dataset which I visualized, and write in python to clean it (about 900+ images is wrong), but the training result is not well, I think maybe something wrong with landmark training process.
